doc/adrs/113_add_support_for_multiple_IObjectService.adoc

```diff
@@ -0,0 +1,84 @@
+= ADR-113 - Add support for multiple IObjectService
+
+== Context
+
+Only one `IObjectService` can be provided in a Sirius Web based application.
+The default implementation (`org.eclipse.sirius.components.emf.services.ObjectService`) is a good default implementation but sometimes we need to customize it based on a specific predicate.
+
+== Decision
+
+* Add new `IObjectServiceDelegate`.
+All Sirius Web applications may provide as many `IObjectServiceDelegate` as they want.
+
+[source,java]
+----
+public interface IObjectServiceDelegate {
+    
+    boolean canHandle(IEditingContext editingContext, Object object);
```

Comment:
You can also remove the editingContext here (see my comment below) since I didn't catch it at first.

Reply:
Done